app/tests/V1/test_views.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import os
+import json
+
+from app import create_app
+
+INCIDENTS_URL = '/red-flags'
+INCIDENT_URL = '/red-flag/1'
+
+class test_IncidentsTestCases (unittest.TestCase):
+    #This represent the Red-flag test case
+    def setUp(self):
+        #test varriables to initialize app
+		self.client = self.app.test_client()
+        self.app = create_app(config_name='testing')
+        self.test_flag = {"flag_id":1, "Title":"Corruption in county finance office", "Comment":"dfffsfsfdsfsdfsdfdsfdfsdffdffdfdfffsdffffdf", "Location":"Thika"}
+        
+    def test_CreateFlag(self):
+        #Test API can create a Red-Flag (POST request)
+        response=self.app.Post(INCIDENTS_URL)
+        result=json.loads(response, data =self.Incidents)
+        self.assertEqual(result["msg"],"success")
+        self.assertEqual(response.status_code,201)
```

Comment:
Please test that the data is what you expect as well, instead of just the status code and the success message.